Competition_Task/Competition_Task/Tests/EducationTests.cs

```diff
@@ -1,12 +1,280 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using Competition_Task.Pages;
+using Competition_Task.TestData;
+using Competition_Task.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using OpenQA.Selenium;
 
 namespace Competition_Task.Tests
 {
-    public class EducationTests
+    [TestFixture]
+    //[Parallelizable(ParallelScope.Fixtures)]
+    public class EducationTests:BaseClass
     {
+        
+       
+        
+            LoginPage LoginPageObj = new LoginPage();
```

Comment:
same comments apply as certification class

Reply:
Done the changes in EducationTests also